task-sdk/src/airflow/sdk/definitions/deadline.py

```diff
@@ -179,6 +176,27 @@ class DeadlineReference:
            deadline.evaluate_with()
     """
 
+    class TYPES:
+        """Collection of DeadlineReference types for type checking."""
+
+        # Deadlines that should be created when the DagRun is created.
+        DAGRUN_CREATED = tuple(
+            {
+                ReferenceModels.DagRunLogicalDateDeadline,
+                ReferenceModels.FixedDatetimeDeadline,
+            }
+        )
```

Comment:
Why not just

```suggestion
        DAGRUN_CREATED = (
            ReferenceModels.DagRunLogicalDateDeadline,
            ReferenceModels.FixedDatetimeDeadline,
        )
```

Same below

Reply:
tuple(set()) enforces unique entries, I did it that was as a safety feature.